src/components/map/NavigationButtons.vue

```diff
@@ -1,12 +1,23 @@
 <script setup lang="ts">
 import type { VcsApp } from '@vcmap/core'
 import { Viewpoint } from '@vcmap/core'
-import { inject } from 'vue'
+import { inject, reactive } from 'vue'
 import IconHome from '../ui/icons/IconHome.vue'
 import UiButton from '../ui/UiButton.vue'
 
 const vcsApp = inject('vcsApp') as VcsApp
 
+const state = reactive({
+  is3D: false,
```

Comment:
it's not a good option, because this value is not `false` by default but depend on the config.
you should have `state.is3D = vcsApp.maps.activeMap.name === 'cesium'` here as well